tests/test_config.py

```diff
@@ -25,6 +26,30 @@
 from gpflow.utilities import to_default_float, to_default_int
 
 
+def test_env_variables():
+    os.environ["GPFLOW_INT"] = "int64"
+    os.environ["GPFLOW_FLOAT"] = "float32"
+    os.environ["GPFLOW_POSITIVE_BIJECTOR"] = "exp"
+    os.environ["GPFLOW_POSITIVE_MINIMUM"] = "1e-3"
+    os.environ["GPFLOW_SUMMARY_FMT"] = "simple"
+    os.environ["GPFLOW_JITTER"] = "1e-2"
```

Comment:
This test relies on all of these actually being different from the default values, is that correct? Would it be sensible to test for two different values each to make sure the test will be independent of external state?

Reply:
done